lib/Populate.ts

```diff
@@ -29,7 +29,7 @@ export function PopulateDocument (this: Document, settings?: PopulateSettings |
 	const {model} = this;
 	const localSettings = settings;
 	const promise = model.schemaForObject(this).then((schema) => {
-		const modelAttributes: any[] = utils.array_flatten(schema.attributes().map((prop) => ({prop, "details": schema.getAttributeTypeDetails(prop)}))).filter((obj) => Array.isArray(obj.details) ? obj.details.some((detail) => detail.name === "Model") : obj.details.name === "Model").map((obj) => obj.prop);
+		const modelAttributes: any[] = utils.array_flatten(schema.attributes().map((prop) => ({prop, "details": schema.getAttributeTypeDetails(prop)}))).filter((obj) => Array.isArray(obj.details) ? obj.details.some((detail) => detail.name === "Model"/* || detail.name === "Model Set"*/) : obj.details.name === "Model" || obj.details.name === "Model Set").map((obj) => obj.prop);
```

Comment:
Remove the commented out part?

Reply:
@BraidenCutforth I should just add a TODO here. Technically I think this needs to be done. But I can't think of how to build a test around it right now. It's such a weird & unique edge case, that I think it's ok for now.

Reply:
```suggestion
		// TODO: uncomment out `/* || detail.name === "Model Set"*/` part and add relevant tests
		const modelAttributes: any[] = utils.array_flatten(schema.attributes().map((prop) => ({prop, "details": schema.getAttributeTypeDetails(prop)}))).filter((obj) => Array.isArray(obj.details) ? obj.details.some((detail) => detail.name === "Model"/* || detail.name === "Model Set"*/) : obj.details.name === "Model" || obj.details.name === "Model Set").map((obj) => obj.prop);
```